sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityTests.java

```diff
@@ -142,6 +159,19 @@ public void deleteUserWithResponse(HttpClient httpClient) {
         assertEquals(204, response.getStatusCode(), "Expect status code to be 204");
     }
 
+    @ParameterizedTest
+    @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    public void deleteUserWithResponseWithNullContext(HttpClient httpClient) {
+        // Arrange
+        CommunicationIdentityClientBuilder builder = getCommunicationIdentityClient(httpClient);
```

Comment:
nit: "get" sounds like it existed before, maybe we should call it "create". Also it creates a builder not a client. We can call it `createClientBuilder` (We are in the CommunicationIdentity test, so I think that sets the context and we do not need to explictly call out what type of clients we are building in the name).

Reply:
Thanks for catching this! This makes a lot of sense